WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -507,5 +527,17 @@ private extension DefaultProductFormTableViewModel {
         static let noPriceWarningTitle =
             NSLocalizedString("Variations without price won’t be shown in your store",
                               comment: "Title of the no price warning row on Product Variation main screen when a variation is enabled without a price")
+
+        // Downloads
+        static let downloadsTitle =
+            NSLocalizedString("Downloadable files",
+                              comment: "Title of the Downloadable Files row on Product main screen for a downloadable product")
```

Comment:
nit: the "downloadable product" doesn't exist (it's not a type as a variable product)
```suggestion
                              comment: "Title of the Downloadable Files row on Product main screen")
```

Reply:
Addressed the issue.